docs/changelog.rst

```diff
@@ -2,8 +2,9 @@
 Changelog
 =========
 
-* :bug:`1740` SNX token and some other token balances should no longer be double counted
-* :feature:`1724` YFI and BAL are now supported as collateral for makerdao vaults
+* :feature:`717` Add support for Uniswap v2 balances, swaps history and LP events history.
```

Comment:
@LefterisJP @kelsos is this all I have to do? I've used the issue number, no matter how many PRs involved.

Reply:
Yes we add the entry on top of the change log. As for the wording it would be nice to follow the same format as the other messages. @LefterisJP is a bit picky about that :smile:

Reply:
Is he picky? really? never seemed to me... :D I've searched for other "protocol additions". As a big feature it is, I could have written "Rotki now supports...".

Reply:
The way he described it to me is "imagine how the user would read it", which I did, but I probably imagined it wrong :smile: 
I guess he can provide an example :smiley:

Reply:
It's fine. If I don't like something I will change it later. Thanksa lot.

And indeed what Kelsos said is correct:

So it should all be in the format:

Users will now be able to .. bla bla. But can change that in another PR. (just look at how other entires are phrased)